packages/nextjs/components/scaffold-eth/ProgressBar.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+
+import { useEffect } from "react";
+import NProgress from "nprogress";
```

Comment:
and you need to remove `next-progressbar` from package.json and to add `nprogress`